src/components/share-overlay/share-overlay.js

```diff
@@ -503,8 +503,9 @@ class ShareOverlay extends Component {
    * @memberof ShareOverlay
    */
   getShareUrl(): string {
-    let url = this.props.config.shareUrl;
-    return this._maybeAddParamsToUrl(url);
+    const {shareUrl, entryId} = this.props.config;
```

Comment:
is the `config` here is the plugin's config? If so, I am not sure that the `entryId` value should come from there.. or maybe i misunderstood. can you please elaborate?

Reply:
this is the plugin config but the entry id is there